make/alerts.mk

```diff
@@ -0,0 +1,25 @@
+export WORKDIR ?= $(shell pwd)
+export IMAGE ?= quay.io/prometheus/prometheus
+export AVAILABILITY_SLO_RULES ?= ${WORKDIR}/examples/alerts/slo-availability.yaml
+export LATENCY_SLO_RULES ?= ${WORKDIR}/examples/alerts/slo-latency.yaml
+export UNIT_TEST_DIR ?= ${WORKDIR}/examples/alerts/tests
+
+
+
+container-runtime-tool:
+	$(eval CONTAINER_RUNTIME_BIN := $(shell if command -v docker &>/dev/null; then \
+                                            echo "docker"; \
+                                        elif command -v podman &>/dev/null; then \
+                                            echo "podman"; \
+                                        else \
+                                            echo "Neither Docker nor Podman is installed. Exiting..."; \
+                                            exit 1; \
+                                        fi))
+
+
+alerts-tests: container-runtime-tool
```

Comment:
would be really nice to have a target that regens the rules.
i.e runs `sloth generate ...`

Reply:
Ooo yeah good shout ill create that